### PR TITLE
Gherkin Scripts for Import VM

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-4183

Problem: Gherkin Scripts Design for Import VM Feature

Solution: These are the gherkin scripts designed covering all the scenarios related to Importing VM

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/virtulization/actions-on-Vm.feature [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    And user will see a Application dropdown
+
+
+@regression
+Scenario: Start VM action on VM: VM-04-TC03
+    Given user is at Developer Perspective
+    And user is having VM on the Topology page
+    When user right clicks on the VM to open the context menu
+    And user clicks on the Start VM
+    Then user will see Stop VM item in context menu
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/virtulization/actions-on-Vm.feature:36]: Can't we make it to one step instead of 3 steps

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/virtulization/actions-on-Vm.feature]: We can but then I need add those option comma separated. I will do that.

**Pull request merged**